lxd/device/nic_bridged.go

```diff
@@ -993,8 +978,16 @@ func (d *nicBridged) removeFilters(m deviceConfig.Device) {
 		return
 	}
 
+	allowedIPs, err = allowedIPNets(map[string]string{
+		"ipv4.address": IPv4Alloc.IP.String(),
+		"ipv6.address": IPv6Alloc.IP.String(),
+	})
+	if err != nil {
+		logger.Error("Failed to get allowed networks from device configuration", log.Ctx{"err": err})
```

Comment:
I think there are some optimisations to be made here for sure. I've had to add another call to `InstanceClearBridgeFilter` to ensure that any global filters are removed. This can happen if there is a parent network that has DHCP disabled, there will then be a global filter applied but nothing in the device config to indicate that.

Reply:
I've removed the extra call so we're back to two calls to `InstanceClearBridgeFilter` i.e. one for static routes and one for dynamic IPs.